scout/server/blueprints/cases/templates/cases/utils.html

```diff
@@ -444,15 +444,23 @@ <h6>No causative variants</h6>
               </div>
               <div class="row">
                 <div class="col-4">
-                  OMIM diagnoses
+                  Diagnoses
                 </div>
                 <div class="col-8">
-                  {% for omim in variant_phenotypes.omim_terms %}
+                  {% for term in variant_phenotypes.omim_terms %}
```

Comment:
Unfortunately these are saved like this in the database. Partial causatives are not used that much so perhaps let's keep the key name like this, knowing that it could be omim and orpha terms